packages/libs/react-ui/src/components/Tabs/Tab.tsx

```diff
@@ -1,30 +1,32 @@
-import classNames from 'classnames';
```

Comment:
I cant tab through the tabs.
I can go through them with the arrow buttons. it feels weird though. first tabbing, and then need to switch to arrows

Reply:
This follows the w3 tabs patterns https://www.w3.org/WAI/ARIA/apg/patterns/tabs/